newVersion.py

```diff
@@ -544,6 +544,13 @@ def add_common_stage_arguments(parser, new_mobile=True):
     parser.add_argument('-t', '--title', dest='title', help='Release title')
     parser.add_argument('--notes', dest='notes', help='Release notes')
 
+def check_node_version():
+    print("Verifying that node version is 16.x")
+    results = subprocess.check_output(['node', '--version'], encoding='UTF-8')
+    match = re.search('^v16\\.', results)
+    if not match:
+        raise Exception("Error: Node 16.x required. You have " + results.rstrip('\n') + ".")
+
```

Comment:
I think this should be future-proof and work with versions > 16 too.

Reply:
Since we include instructions to use Node 16 in the samples, I disagree. Even if it's ok for users to use Node > 16 (which I actually don't know), we still want our published packages to use Node 16 during publishing. This script is only used for publishing our packages, and we want to consistently use Node 16 for that.

Reply:
In other words, we may decide that the instructions can say to use Node 16 or above. However, even if that happens, we still want to consistently publish our packages using Node 16. (Until such time in the future that we move to a newer Node, at which point we consistently use that one.)

Reply:
OK, that's fine.